addons/fleet/views/fleet_vehicle_cost_views.xml

```diff
@@ -95,27 +95,16 @@
         <field name="model">fleet.vehicle.log.contract</field>
         <field name="arch" type="xml">
             <kanban class="o_kanban_mobile" sample="1">
-                <field name="activity_state"/>
                 <progressbar field="activity_state" colors='{"planned": "success", "today": "warning", "overdue": "danger"}'/>
                 <templates>
-                    <t t-name="kanban-box">
-                        <div t-attf-class="oe_kanban_global_click">
-                            <div>
-                                <strong>
-                                    <field name="vehicle_id" widget="res_partner_many2one"/>
-                                    <span class="float-end badge text-bg-secondary">
-                                        <field name="state"/>
-                                    </span>
-                                </strong>
-                            </div>
-                            <div>
-                                <t t-if="luxon.DateTime.fromISO(record.expiration_date.raw_value) &lt; luxon.DateTime.local()" t-set="expiration_class" t-value="'oe_kanban_text_red'"/>
-                                <span t-att-class="expiration_class"><field name="start_date"/> - <field name="expiration_date"/></span>
-                            </div>
-                            <div>
-                                <field name="insurer_id" widget="res_partner_many2one"/>
-                            </div>
+                    <t t-name="kanban-card">
+                        <div class="d-flex justify-content-between align-items-center">
+                            <field name="vehicle_id" class="fw-bold" widget="res_partner_many2one"/>
+                            <field class="float-end badge text-bg-secondary" name="state"/>
```

Comment:
I would use ms-auto instead of float-end (to test though)

Reply:
i think i have used justify-content-between so i not need to use float-end or ms-auto